src/Raven.Client/Documents/Operations/AI/GenAiConfiguration.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Raven.Client.Documents.Operations.ETL;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.AI;
+
+public class GenAiConfiguration : EtlConfiguration<AiConnectionString>
+{
+
+    [JsonDeserializationIgnore]
+    [JsonIgnore]
+    public AiConnectorType AiConnectorType => Connection?.GetActiveProvider() ?? AiConnectorType.None;
+
+    public override string GetDestination() => Name;
+    public override string GetDefaultTaskName() => Name;
+
+    public string Identifier { get; set; }
+    public string Collection { get; set; }
+    
+    public override EtlType EtlType => EtlType.GenAi;
+    public override bool UsingEncryptedCommunicationChannel() => Connection?.UsingEncryptedCommunicationChannel() ?? false;
+
+    public string GenerateIdentifier() => EmbeddingsGenerationConfiguration.GenerateIdentifier(Name);
```

Comment:
shouldn't be here? check how this is handled in EmbeddingsGenerationConfiguration